sections/video.liquid

```diff
@@ -60,6 +60,7 @@
         id="Deferred-Poster-Modal-{{ section.settings.video_url.id }}"
         class="video-section__poster media deferred-media__poster media--landscape"
         type="button"
+        aria-label="Load video: {{ video.alt }}"
```

Comment:
So in the theme we don't use many english strings directly in the HTML as it's information that's going to need to be translated in whatever language the merchant is using for their storefront. 

We have a specific way to deal with translation internally so in this case here, it can be hard to understand the way to go about it. It would need to be: 
```suggestion
        aria-label="{{ 'sections.video.load_video' | t: description: section.settings.description | escape }}"
```

And for the video in the collage section, you could probably use: 
```
aria-label="{{ 'sections.video.load_video' | t: description: block.settings.description | escape }}"
```

On the `<button>` element.

Reply:
Added to the video button in the collage section.